libs/dal/src/repositories/subscriber/subscriber.schema.ts

```diff
@@ -27,8 +27,8 @@ const subscriberSchema = new Schema(
   schemaOptions
 );
 
-subscriberSchema.index({ _environmentId: 1, userId: 1 });
 subscriberSchema.index({ _environmentId: 1, subscriberId: 1 });
+subscriberSchema.index({ _environmentId: 1, _organizationId: 1, subscriberId: 1 });
```

Comment:
```suggestion
```

I actually think that this one is not needed, since the above index is enough specific to find the query. Mongo is smart enough to use the above one even you perform a search with organizationId.